test/integration/features/config.feature

```diff
@@ -122,7 +122,7 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
        And executing "crc config set skip-check-crc-network-active true" succeeds
        # Start CRC
        Then starting CRC with default bundle and default hypervisor fails
-       And stdout contains "Network not found: no network with matching name 'crc'"
+       And stderr contains "network crc not found"
```

Comment:
I'd go with `no network with matching name` as `network crc not found` is not part of the error message. Also, there's a typo in that error message in `Use 'crc setup' for define the network`. Needs 'to' instead of 'for'. But I guess that doesn't belong into this PR...